Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -24,56 +24,53 @@ class MyTestInParallel_2(EditorParallelTest):
 """
 from __future__ import annotations
 
-import tempfile
-
-import pytest
-import _pytest.python
-import _pytest.outcomes
-from _pytest.skipping import pytest_runtest_setup as skipping_pytest_runtest_setup
-
 import abc
 import functools
 import inspect
 import json
 import logging
 import math
 import os
+import pytest
 import re
+import tempfile
 import threading
 import types
 import warnings
 
+import _pytest.python
+import _pytest.outcomes
+from _pytest.skipping import pytest_runtest_setup as skip_pytest_runtest_setup
+
 import ly_test_tools.environment.process_utils as process_utils
 import ly_test_tools.o3de.editor_test_utils as editor_utils
-import ly_test_tools._internal.pytest_plugin.test_tools_fixtures
-
 from ly_test_tools.o3de.asset_processor import AssetProcessor
 from ly_test_tools.launchers.exceptions import WaitTimeoutError
 
-# This file contains ready-to-use test functions which are not actual tests, avoid pytest collection
-__test__ = False
+__test__ = False  # This file contains ready-to-use test functions which are not actual tests, avoid pytest collection
 
 logger = logging.getLogger(__name__)
 
 
-class EditorTestBase(abc.ABC):
+class EditorTest(abc.ABC):
```

Comment:
While this is marked as an Abstract Class, this designation will only provide a hint that users should not be directly instantiate it. Since it has no `@abstractmethod`s it will not raise when directly constructed. I debated refactoring the child classes to help enforce this and avoid user error. However since most specializations are also method-less data containers I chose to leave that for another time.